### ISSUE TITLE
[object object] bug: complex flounder config options incorrectly modified on init

### ISSUE THREAD

**Issue by kevin-krug-sociomantic** [sociomantic-tsunami/flounder#77]
## Current 

As of ```0.7.0``` the following complex options are not currently rendered: 

```
new Flounder(<somedomelement>, 
{
 data:
   [ {
      "header": "header1",
      "data": [
        {
          "text": "option1",
          "value": "option1",
          "index": 0
        },
        {
          "text": "option2",
          "value": "option3",
          "index": 1
        },
        {
          "text": "option3",
          "value": "option3",
          "index": 2
        }
      ]
    },
    {
      "text": "probably the string you want to see",
      "value": "return value",
      "description": "a longer description of this element",
      "index": 3
    },
    {
      "header": "header2",
      "data": [
        {
          "text": "option1",
          "value": "option1",
          "index": 4
        },
        {
          "text": "option2",
          "value": "option2",
          "index": 5
        }
      ]
    }
  ]
}
```

yields: 

![screen shot 2016-04-18 at 15 20 05](https://cloud.githubusercontent.com/assets/6839359/14605546/5df3061a-0579-11e6-98e1-c70b9dfac757.png)

checking the resulting flounder object: 

![screen shot 2016-04-18 at 15 22 51](https://cloud.githubusercontent.com/assets/6839359/14606032/aa6ec720-057b-11e6-8ce2-e5438addb287.png)

## Wanted 

flounder should correctly render the options

**Comment by mousemke** [comment]
can you confirm your version?  This was specifically fixed in 0.7.1

**Comment by kevin-krug-sociomantic** [comment]
@mousemke yes, sorry i was on ```0.7.0```. this parse issue seems fixed on ```0.7.1``` ! :+1:
